### ISSUE TITLE
Won't work for me on Chrome

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#597]
```
What's the problem?
I add my blog details (I have done so in different ways) and Scribe Fire 
accepts it fine but it does not come up on my list of Blogs (it's the only one 
I use anyhow). In other words, I sadly cannot use Scribe Fire :-(




What browser are you using?
Chrome





What version of ScribeFire are you running?
4.1

```

-----
Original issue reported on code.google.com by `caroline...@gmail.com` on 5 Apr 2012 at 2:19

**Comment by GoogleCodeExporter** [comment]
```
I have the same problem with firefox; get the "embarrassing" dialog.
```

-----
Original comment by `Magna...@gmail.com` on 24 Oct 2012 at 10:04

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in version 4.2 for Chrome, but if you still have an issue, 
please let us know which blog type you're using.
```

-----
Original comment by `scribefi...@gmail.com` on 29 Aug 2013 at 3:39
* Changed state: **Fixed**
